src/search.js

```diff
@@ -27,7 +32,7 @@ export function addNormalizedDescription(propertyName, objects) {
   return map(
     object => assoc(
       'normalizedDescription',
-      has(propertyName, object)
+      object[propertyName]
```

Comment:
My bad, I didn't make the link with the new API. Thanks a lot for the clarification @fpagnoux!  :)